cypress/e2e/resource_spec.js

```diff
@@ -88,6 +106,7 @@ it('Test edit dataset name languages', function(){
 });
 
 
+//it.only('Finnish UI elements', function() {
```

Comment:
Could be removed.

Reply:
Resolved in a7c3f51ce5f44784a526999e153cc0c222287623